packages/oracle/data_stream/database_audit/_dev/test/pipeline/test-oracle-database-audit.log-expected.json

```
@@ -1,15 +1,15 @@
 {
     "expected": [
         {
-            "@timestamp": "2020-10-07T15:58:08.000Z",
+            "@timestamp": "2020-10-21T15:58:08.000Z",
             "ecs": {
                 "version": "8.0.0"
             },
             "event": {
                 "action": "database_audit",
                 "category": "database",
                 "kind": "event",
-                "original": "Wed Oct  7 11:58:08 2020 -04:00\nLENGTH : '392'\nACTION :[151] 'select /*+ opt_param('parallel_execution_enabled',\n                                   'false') EXEC_FROM_DBMS_XPLAN */ * from gv$all_sql_plan where 1=0'\nDATABASE USER:[1] '/'\nPRIVILEGE :[4] 'NONE'\nCLIENT USER:[0] ''\nCLIENT TERMINAL:[7] 'UNKNOWN'\nSTATUS:[1] '0'\nDBID:[10] '2824230686'\nSESSIONID:[1] '0'\nUSERHOST:[13] 'testlab.local'\nCLIENT ADDRESS:[0] ''\nACTION NUMBER:[1] '3'\n",
+                "original": "Wed Oct  21 11:58:08 2020 -04:00\nLENGTH : '392'\nACTION :[151] 'select /*+ opt_param('parallel_execution_enabled',\n                                   'false') EXEC_FROM_DBMS_XPLAN */ * from gv$all_sql_plan where 1=0'\nDATABASE USER:[1] '/'\nPRIVILEGE :[4] 'NONE'\nCLIENT USER:[0] ''\nCLIENT TERMINAL:[7] 'UNKNOWN'\nSTATUS:[1] '0'\nDBID:[10] '2824230686'\nSESSIONID:[1] '0'\nUSERHOST:[13] 'testlab.local'\nCLIENT ADDRESS:[0] ''\nACTION NUMBER:[1] '3'\n",
```

This file should be modified or regenerated too.
```suggestion
                "original": "Wed Oct 21 11:58:08 2020 -04:00\nLENGTH : '392'\nACTION :[151] 'select /*+ opt_param('parallel_execution_enabled',\n                                   'false') EXEC_FROM_DBMS_XPLAN */ * from gv$all_sql_plan where 1=0'\nDATABASE USER:[1] '/'\nPRIVILEGE :[4] 'NONE'\nCLIENT USER:[0] ''\nCLIENT TERMINAL:[7] 'UNKNOWN'\nSTATUS:[1] '0'\nDBID:[10] '2824230686'\nSESSIONID:[1] '0'\nUSERHOST:[13] 'testlab.local'\nCLIENT ADDRESS:[0] ''\nACTION NUMBER:[1] '3'\n",
```